aspnet-server-1/src/IO.Swagger/Controllers/DefaultApi.cs

```diff
@@ -0,0 +1,167 @@
+/*
+ * Walking Skeleton API
+ *
+ * This is the API for the Walking Skeleton.
+ *
+ * OpenAPI spec version: 1
+ * 
+ * Generated by: https://github.com/swagger-api/swagger-codegen.git
+ */
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using Swashbuckle.AspNetCore.SwaggerGen;
+using Newtonsoft.Json;
+using System.ComponentModel.DataAnnotations;
+using IO.Swagger.Attributes;
+
+using Microsoft.AspNetCore.Authorization;
+using IO.Swagger.Models;
+using MongoDB.Driver;
+using MongoDB.Bson;
+
+namespace IO.Swagger.Controllers
+{ 
+    /// <summary>
+    /// 
+    /// </summary>
+    [ApiController]
+    public class DefaultApiController : ControllerBase
+    { 
+        /// <summary>
+        /// deletes a location
+        /// </summary>
+        /// <param name="deviceId">The ID of the device.</param>
+        /// <response code="200">location deleted successfully</response>
+        /// <response code="400">deletion of location went wrong</response>
+        /// <response code="404">no location associated with devive ID</response>
+        [HttpDelete]
+        [Route("/Location/{deviceId}")]
+        [ValidateModelState]
+        [SwaggerOperation("DeleteLocation")]
+        public virtual IActionResult DeleteLocation([FromRoute][Required]int? deviceId)
+        {
+            // TODO: Initiate client elsewhere
+            MongoClient dbClient = new MongoClient("mongodb://admin:mongo801@130.225.57.95:27017");
```

Comment:
Make a String const for "mongodb://admin:mongo801@130.225.57.95:27017"

Reply:
It is duplicated 4 times :D

Reply:
Thought about it aswell, so I will do it :D